File: GGAndroid_2.0/GGDemo/src/main/java/io/ganguo/demo/entity/UserEntity.java
package io.ganguo.demo.entity;

import com.google.gson.annotations.SerializedName;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.Accessors;

/**
 * Created by Tony on 10/6/15.
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
@Accessors(chain = true)
@org.parceler.Parcel
public class UserEntity {
    int id;
    String name;
    @SerializedName("api_token")
    String apiToken;

}

File: GGAndroid_3.0/main/app-mvvm-v1-sample/src/main/java/io/ganguo/demo/entity/UserEntity.kt
package io.ganguo.demo.entity

import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

/**
 * Created by Tony on 10/6/15.
 */
class UserEntity(
        var id: Int = 0,
        var name: String? = null,
        @SerializedName("api_token")
        var apiToken: String? = null
)
